app/src/userpages/components/ProfilePage/DeleteAccount/index.jsx

```diff
@@ -0,0 +1,95 @@
+// @flow
+
+import React from 'react'
+import { connect } from 'react-redux'
+import { Translate, I18n } from 'react-redux-i18n'
+import { Button } from 'reactstrap'
+
+import DeleteAccountDialog from './DeleteAccountDialog'
+import { deleteUserAccount } from '$shared/modules/user/actions'
+import { selectDeletingUserAccount } from '$shared/modules/user/selectors'
+import type { StoreState } from '$shared/flowtype/store-state'
+
+import styles from './deleteAccount.pcss'
+
+type State = {
+    modalOpen: boolean,
+}
+
+type StateProps = {
+    deletingUserAccount: boolean,
+}
+type DispatchProps = {
+    deleteAccount: () => Promise<null>,
+}
+
+type Props = StateProps & DispatchProps
+
+class DeleteAccount extends React.Component<Props, State> {
+    state = {
+        modalOpen: false,
+    }
+
+    openModal = () => {
+        this.setState({
+            modalOpen: true,
+        })
+    }
+
+    onClose = () => {
+        this.setState({
+            modalOpen: false,
+        })
+    }
+
+    onSave = () => {
+        this.props.deleteAccount()
+            .then(() => {
+                this.onClose()
+            }, (error) => {
+                alert(error.message) // eslint-disable-line no-alert
+            })
+    }
+
+    render() {
+        const { modalOpen } = this.state
+        const { deletingUserAccount } = this.props
+
+        return (
+            <div>
+                <div>
+                    <Translate value="userpages.profilePage.deleteAccount.description" />
+                </div>
```

Comment:
If the intention is to get this `display: block`, perhaps you could use `<Translate tag='div' />` instead of wrapping this in an anonymous `div`? One less level of nesting. https://github.com/JSxMachina/react-i18nify#translate

Reply:
Yes, that was the intention. Good catch!